common/src/main/java/com/wynntils/core/features/UserFeature.java

```diff
@@ -20,7 +22,6 @@ public final void updateConfigOption(ConfigHolder configHolder) {
         // if user toggle was changed, enable/disable feature accordingly
         if (configHolder.getFieldName().equals("userEnabled")) {
             // If this was changed before init, do not try to toggle
```

Comment:
TODO: Update this comment to reflect uninitalized state, and try to make it more obvious